apl-core/src/test/java/com/apollocurrency/aplwallet/apl/core/dgs/DGSServiceTest.java

```diff
@@ -85,18 +78,26 @@
     Blockchain blockchain = mock(Blockchain.class);
     @WeldSetup
     public WeldInitiator weld = WeldInitiator.from(
-            PropertiesHolder.class, BlockchainConfig.class,
+            PropertiesHolder.class,
             EpochTime.class,
             GlobalSyncImpl.class,
             FullTextConfigImpl.class,
             DGSPublicFeedbackTable.class,
             DGSFeedbackTable.class,
             DGSGoodsTable.class,
             DGSTagTable.class,
-            AccountTable.class,
             DGSPurchaseTable.class,
             DGSServiceImpl.class,
-            DerivedDbTablesRegistryImpl.class)
+            DerivedDbTablesRegistryImpl.class,
```

Comment:
Looks like a copy-paste from previous tests, analyze, which tests require accountService and remove redundant dependncies

Reply:
partially done, this test depends on the account service/table behavior.